frontend/index.html

```
@@ -1,14 +1,26 @@
 <!DOCTYPE html>
 <html lang="">
```

HTMLHint (1.9.2)

[warning] The lang attribute of <html> element must have a value.

(html-lang-require)